tests/test_sesv2/test_server.py

```diff
@@ -0,0 +1,12 @@
+"""Test different server responses."""
+
+import moto.server as server
+
+
+def test_sesv2_list():
+    backend = server.create_backend_app("sesv2")
+    test_client = backend.test_client()
+
+    resp = test_client.post("/v2/email/contact-lists/test/contacts")
```

Comment:
```suggestion
    resp = test_client.get("/v2/email/contact-lists/test/contacts")
```
The POST-method resolves to `create_contact`, which isn't implemented yet. The GET-method will resolve to `list_contacts`.

Note that this particular test still failed for me due to an bug in Moto itself - but it should pass after #6266 is merged.